uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/CamelCatalogUtils.java

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.fuse.reddeer;
+
+import java.io.IOException;
+
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.reddeer.common.logging.Logger;
+
+/**
+ * Structure for using different versions of Camel catalog - implemented only for 'components'
+ * 
+ * @author djelinek
+ */
+public class CamelCatalogUtils {
+
+	private static Logger log = Logger.getLogger(CamelCatalogUtils.class);
+
+	private static final String ROOT_PATH = "/org/apache/camel/catalog/";
+
+	private static String COMPONENTS = ROOT_PATH + "components";
+
+	private static String COMPONENTS_LIST = ROOT_PATH + "components.properties";
+
+	public CamelCatalogUtils(String path) {
+		COMPONENTS = path + COMPONENTS;
+		COMPONENTS_LIST = path + COMPONENTS_LIST;
+	}
+
+	public CamelCatalogUtils(IPath path) {
+		this(path.toString());
+	}
+
+	/**
+	 * Returns all available components
+	 * 
+	 * @return String
+	 */
+	public String Components() {
+		try {
+			return FileUtils.getFileContent(COMPONENTS_LIST);
+		} catch (IOException e) {
+			log.error("Resource missing: can't find a failing test case to copy (" + COMPONENTS_LIST + ")!");
+		}
+		log.info("Text in active text editor was replaced with content of the file: " + COMPONENTS_LIST + ".");
+		return "";
+	}
+
+	/**
+	 * Checks if component is in CamelCatalog
+	 * 
+	 * @param name
+	 *            (name of Component)
+	 * @return true/false
+	 */
+	public boolean isExistComponent(String name) {
+		return isExist(name, Components());
+	}
+
+	private boolean isExist(String name, String componentList) {
+		if (componentList.contains(name.toLowerCase().replaceAll("\\s+", "")))
```

Comment:
I think name.toLowerCase() can be extracted as local variable, it will shorten the line